docs/deployment/pulumi/aws.md

```diff
@@ -0,0 +1,188 @@
+        "ecr:DeleteRepository",
+        "ecr:GetRegistryScanningConfiguration",
+        "ecr:CompleteLayerUpload",
+        "ecr:TagResource",
+        "ecr:DescribeRepositories",
+        "ecr:DeleteRepositoryPolicy",
+        "ecr:BatchCheckLayerAvailability",
+        "ecr:GetLifecyclePolicy",
+        "ecr:GetRegistryPolicy",
+        "ecr:PutLifecyclePolicy",
+        "ecr:DescribeImageScanFindings",
+        "ecr:GetLifecyclePolicyPreview",
+        "ecr:CreateRepository",
+        "ecr:DescribeRegistry",
+        "ecr:GetDownloadUrlForLayer",
+        "ecr:GetAuthorizationToken",
+        "ecr:DeleteLifecyclePolicy",
+        "ecr:PutImage",
+        "ecr:UntagResource",
+        "ecr:SetRepositoryPolicy",
+        "ecr:BatchGetImage",
+        "ecr:InitiateLayerUpload",
+        "ecr:GetRepositoryPolicy",
+        "lightsail:CreateContainerService",
+        "lightsail:GetKeyPair",
+        "lightsail:GetContainerServiceDeployments",
+        "lightsail:CreateContainerServiceRegistryLogin",
+        "lightsail:GetContainerImages",
+        "lightsail:UntagResource",
+        "lightsail:RegisterContainerImage",
+        "lightsail:GetContainerServices",
+        "lightsail:GetContainerServicePowers",
+        "lightsail:GetKeyPairs",
+        "lightsail:CreateContainerServiceDeployment",
+        "lightsail:GetContainerServiceMetricData",
+        "lightsail:GetContainerAPIMetadata",
+        "lightsail:DeleteContainerService",
+        "lightsail:GetContainerLog",
+        "lightsail:TagResource"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "Statement1",
+      "Effect": "Allow",
+      "Action": [],
+      "Resource": []
+    }
+  ]
+}
+```
+
+Then click on `Next` and give the policy an name and a description of your liking. Afterwards, click on `Create policy`.
+
+- Navigate back to the user creation window and press on the refresh button and search for the policy you just created. Now, create the user.
+- Now you will be redirected to all users, click on the user you just created and click on `Security credentials`
+- Scroll below and click on `Create access key`
+- Choose `Command Line Interface (CLI)`
+- Now export the following values
+
+```bash
+$ export AWS_ACCESS_KEY_ID=... (first value)
+$ export AWS_SECRET_ACCESS_KEY=.... (second secret value)
+```
+
+## Configuring the Pulumi CLI
+
+Second, install the [Pulumi CLI](https://www.pulumi.com/docs/get-started/install/).
+
+Then we need to execute the following commands, to set Pulumi up:
+
+```bash
+$ pulumi login --local
+$ export PULUMI_CONFIG_PASSPHRASE="<your-secret>"
+```
+
+By using `pulumi login --local` we are making sure that Pulumi stores our state on the local file disk. The environment variable `PULUMI_CONFIG_PASSPHRASE` is used by Pulumi to generate a unique key for your stack
+
+## Deploying your instance on Lightsail
+
+:::warning
+
+Make sure that Docker is running before you start with this section.
```

Comment:
Maybe we should link to [Docker](https://www.docker.com/) here, as not everyone may be aware of what it is.